rest-api-spec/src/main/resources/rest-api-spec/test/cat.aliases/10_basic.yaml

```diff
@@ -127,6 +127,51 @@
       $body: / (^|\n)test_2 .+ \n/
 
 ---
+"Multiple alias names":
+
+  - skip:
+      version: " - 5.99.99"
+      reason:  multiple aliases are supported only from 6.0.0 on
+
+  - do:
+        indices.create:
+            index: test
+
+  - do:
+        indices.create:
+            index: test2
+  - do:
+        indices.create:
+            index: test3
+
+  - do:
+        indices.put_alias:
+            index: test
+            name: foo
+
+  - do:
+        indices.put_alias:
+            index: test2
+            name: bar
+  - do:
+        indices.put_alias:
+            index: test3
+            name: baz
+
+  - do:
+        cat.aliases:
+            name: foo,bar
+
+  - match:
+      $body: /(^|\n)foo\s+test /
+
+  - match:
+      $body: /(^|\n)bar\s+test2 /
+
+  - match:
+      $body: /(^|\n)[^bar\s+test3] /
```

Comment:
I think we can and should make a stronger assertion here. We can request only the `alias` and `index` fields, request that the cat API sort the output on the `index` field, and then assert the *exact* expected output:

```
foo test
bar test2
```